experiments/utils.py

```diff
@@ -141,6 +141,11 @@ def get_alternative(self, experiment_name, request=None):
         if experiment and not disabled:
             if experiment.is_displaying_alternatives():
                 alternative = self._get_enrollment(experiment)
+                if request:
+                    request.experiments_exposure = {
+                        'experiment_name': experiment_name,
+                        'experiment_variant': alternative
```

Comment:
This logic should be moved down into `_get_enrollment()` or duplicated in `enroll()` because `enroll()` also calls `_get_enrollment` and returns an alternative, see line 115 in this file.